sys/include/net/gnrc/netif/conf.h

```diff
@@ -111,7 +123,8 @@ extern "C" {
  * Default: 2 (1 link-local + 1 global address)
```

Comment:
Maybe amend the doc here as well.

Reply:
Would this work?

```suggestion
 * Default: 2 (1 link-local + 1 global address) + @ref CONFIG_GNRC_NETIF_IPV6_ADDRS_NUMOF
```

Reply:
I think you meant

```suggestion
 * Default: 2 (1 link-local + 1 global address) + @ref GNRC_NETIF_IPV6_DHCPV6_IA_NA_ADDR
```

right?

If so, it would, but since the current wording is very top-down, I personally would prefer:

```suggestion
```
```suggestion
 * Default: 2 (1 link-local + 1 global address) + any additional address via configuration protocol (e.g. DHCPv6 leases).
```

Reply:
> I think you meant
> 
> right?

Oh, yes, thank you!

> 
> If so, it would, but since the current wording is very top-down, I personally would prefer:

Looks very good, I'll commit this change :)